android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java

```diff
@@ -164,6 +168,18 @@ public void configure(TapGestureHandler handler, ReadableMap config) {
       if (config.hasKey(KEY_TAP_MAX_DELAY_MS)) {
         handler.setMaxDelayMs(config.getInt(KEY_TAP_MAX_DELAY_MS));
       }
+      if (config.hasKey(KEY_TAP_MAX_DELTA_X)) {
+        handler.setMaxDx(config.getInt(KEY_TAP_MAX_DELTA_X));
```

Comment:
For params that refer to distances we need to transform from DP units that react is using into pixels. Please take a look few lines down where we extract `KEY_LONG_PRESS_MAX_DIST` for the long press handler:

```
if (config.hasKey(KEY_LONG_PRESS_MAX_DIST)) {
    handler.setMaxDist(PixelUtil.toPixelFromDIP(config.getDouble(KEY_LONG_PRESS_MAX_DIST)));
}
```

Also note that we should use `getDouble` not `getInt`